docs/index.rst

```diff
@@ -0,0 +1,12 @@
+The Shotgun Desktop Client Framework
+==================================================
+
+The Shotgun Desktop Client Framework contains an implementation of a remote client that
+has the capabilities to communicate with Shotgun Create. This client takes a Shotgun
+connection and makes all operations to succeed at the Shotgun Create's desktop client handshake.
+
+.. automodule:: create_client
+    :members:
+
+.. autoclass:: CreateClient
+    :members:
```

Comment:
I would like to see under this a section with the list of commands that are currently supported by Create, so our clients can know what is possible.